---
layout: base
title:  'Universal Dependencies --- Syntactic validation'
udver:  '2'
---

Regenerated <time class="timeago" datetime="2017-02-08T14:13:51Z">2017-02-08T14:13:51 zulu</time>

Traceback (most recent call last):
  File "docs-pages-source/gen_index/syn_validation_run.py", line 267, in <module>
    del lang_results[lw]
KeyError: 'UD_Polish'
